EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/DependenciesPanel.java

```diff
@@ -226,6 +235,30 @@ public void onSubmit(final AjaxRequestTarget target, final Form form) {
     }
 
     /**
+     * Create and add a link to delete the selected attribute dependency.
+     */
+    private void addDeleteDependencyButton() {
+        AjaxLink deleteDependencyButton = new AjaxLink<Void>("deleteDependencyButton") {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This class has 6 parents which is greater than 5 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)